docs/docs/guides/01-basics/03-http/07-file-uploads.md

```diff
@@ -125,6 +125,81 @@ await avatar.move(Application.tmpPath('uploads'), {
 })
 ```
 
+## Streaming Files
+
+## Disable auto-processing
+
+First, disable file auto-processing for your upload routes via the `config/bodyparser.ts` file:
+
+```ts
+processManually: ['/upload']
+```
+
+Alternatively, you can do this for all routes, like so:
+
+```ts
+autoProcess: false
+```
+
+### Single File
+
+```ts
+request.multipart.onFile('file', {}, async (file) => {
+  AWS.config.update({
```

Comment:
Shouldn't this be happening once at the application boot level vs doing it on every request?